### PR TITLE
 _ApiVersion -> ApiVersion

### PR DESCRIPTION
**Pull request by richardm-stripe**
* Changes _ApiVersion to ApiVersion, as I think it's legitimate for users to want to ask what the pinned version of the library is.
* ~Updates the version.~ (apparently release script does this automatically)

### CODE REVIEW

**Comment by remi-stripe** [comment]
Can we fix the title as the API version number is wrong

File: stripe/api_version.py [hunk 1]
```diff
@@ -2,5 +2,5 @@
 # File generated from our OpenAPI spec
 
 
-class _ApiVersion:
-    CURRENT = "2022-11-15"
+class ApiVersion:
```
Comment (pakrym-stripe) [inline stripe/api_version.py:5]: Changes _ApiVersion to ApiVersion, as I think it's legitimate for users to want to ask what the pinned version of the library is.

They have `Stripe.api_version`  for that.

**Review by pakrym-stripe — CHANGES_REQUESTED** [review]
Do we need two ways to get api_version?

File: stripe/api_version.py (hunk 1 above)
Comment (richardm-stripe) [reply stripe/api_version.py:5]: Stripe.api_version is the version *currently in use*. It *defaults* to ApiVersion.CURRENT, but is not a constant describing the library's pinned version.

Comment (richardm-stripe) [reply stripe/api_version.py:5]: Also FWIW this is public in every other library.

Comment (pakrym-stripe) [reply stripe/api_version.py:5]: We also don't have a similar feature in node.

**Pull request closed without merging**